source/docs/api-reference/api-usage/swerve/swerve-overview.rst

```diff
@@ -0,0 +1,66 @@
+Swerve Overview
+===============
+
+New in 2024 is a high performance swerve framework. This framework simplifies the boilerplate necessary for swerve and maximizes performance.
+
+.. important:: This section focuses on utilizing the Swerve API and configuring it correctly. Tuner X supports a swerve project creator that greatly simplifies this process and removes common error cases.
+
+- Small API surface, easily debuggable
+
+  * Build robot characteristics using ``SwerveModuleConstants`` (Java) and ``SwerveDrivetrainConstants``.
+  * Integrates cleanly into WPILib `commandbased <https://docs.wpilib.org/en/stable/docs/software/commandbased/index.html>`__ framework via ``CommandSwerveDrivetrain`` (Java).
+  * Telemetrize directly in the odometry loop using the ``registerTelemetry()`` (Java) lambda.
+  * Supports handling the swerve state via ``SwerveRequest`` (Java).
+
+    * Supports robot-centric, field-centric and field-centric facing angle.
+    * Supports common scenarios such as X module (point all modules toward the center of the robot).
+
+- Simulation
+
+  * Swerve simulation focuses on usability, and as such isn't perfectly accurate to a real robot.
+  * Simply call ``updateSimState`` (Java) in ``simulationPeriodic()`` (`Java <https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj/IterativeRobotBase.html#simulationPeriodic()>`__).
```

Comment:
Maybe make a note that this is done already if you're using the example `CommandSwerveDrivetrain` class